src/bootsampling.jl

```diff
@@ -103,6 +103,24 @@ end
 
 ExactSampling() = ExactSampling(0)
 
+"""
+[Maximum Entropy Sampling](https://cran.r-project.org/web/packages/meboot/vignettes/meboot.pdf)
+
+```julia
+MaximumEntropySampling()
+MaximumEntropySampling(100)
+maximumEntropySampling(100, MaximumEntropyCache())
+```
+
+NOTE: Implementation based off [pymeboot](https://github.com/kirajcg/pymeboot) as the original
+[R package](https://cran.r-project.org/web/packages/meboot/index.html) is GPL licensed.
+"""
+type MaximumEntropySampling <: BootstrapSampling
+    nrun::Int
+    cache::MaximumEntropyCache
+end
+
+MaximumEntropySampling(nrun=0) = MaximumEntropySampling(nrun, MaximumEntropyCache())
```

Comment:
I wouldn't give `nrun` a default value since this is really something the user has to decide based on the input data and the analysis (and `0` doesn't do anything meaningful anyway).